### ISSUE TITLE
[Mod-Escalated] Chat needs to stop auto-scrolling to the bottom - it makes moderation impossible on mobile.

### ISSUE THREAD

**Issue by QuinnBracelen** [mchineboy/treesradio-frontend#38]
Especially when chat is busy, or when trying to moderate from mobile, it's damn near impossible to catch up on older chat messages - making it hard to know the context for conversations and plays.

Going forward, the chat window should remain wherever the user scrolls to.
- If this is the top of chat, then it auto-scrolls as the oldest message leaves the chat.
- If this is not the bottom of the chat, then when a new chat message enters the chat there should be some sort of indicator/visual-flourish to show the user there's additional content below (like a flash, badge # for how many new chats have appeared since they scrolled, change the background color of the chat entry window, idk).

Ideally, we would also have a few additional features added too:
- Jump to top of chat button
- Jump to bottom of chat button
